index.html

```diff
@@ -75,5 +75,251 @@ <h4>LETS CONNECT</h4>
                     </div>
               </div>
        </section> 
+       <section class="grid-container">
+             <div class="project-item">
+                <div class="project-image">
+                  <img src="./assets/first-image.jpg" alt="first project image"/>
+                </div>
+                <div class="project-details">
+                  <div>
+                    <h1>Tonic</h1>
+                  </div>
+                  <div class="canopy-para">
+                    <p><span>CANOPY</span>  <img src="./assets/Counter.png" alt="a picture dot">  Back end dev  <img src="./assets/Counter.png" alt="a picture dot"> 2015</p>
+                  </div>
+                  <div class="mini-description">
+                    <p>
+                      A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    </p>
+                  </div>
+                  <div class="language-tags">
+                     <ul>
+                      <div>
+                        <li>html</li>
+                      </div>
+                      <div>
+                        <li>css</li>
+                      </div>
+                      <div>
+                        <li>javaScript</li>
+                      </div>
+                     </ul>
+                  </div>
+                  <div class="project-detail-btn">
+                    <button>See Project</button>
+                  </div>
+                </div>
+             </div>
+             <div class="project-item">
+                <div class="project-image">
+                  <img src="./assets/second-image.png" alt="second project image"/>
+                </div>
+                <div>
+                  <div class="project-details">
+                    <div>
+                      <h1>Multi-Post <br /> Stories</h1>
+                    </div>
+                    <div class="canopy-para">
+                      <p><span>CANOPY</span> <img src="./assets/Counter.png" alt="a picture dot"> Back end dev <img src="./assets/Counter.png" alt="a picture dot"> 2015</p>
+                    </div>
+                    <div class="mini-description">
+                      <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required.
+                      </p>
+                    </div>
+                    <div class="language-tags">
+                      <ul>
+                        <div>
+                          <li>html</li>
+                        </div>
+                        <div>
+                          <li>css</li>
+                        </div>
+                        <div>
+                          <li>javaScript</li>
+                        </div>
+                      </ul>
+                    </div>
+                    <div class="project-detail-btn">
+                      <button>See Project</button>
```

Comment:
- [ ] Kindly add these interactions with your buttons as the Figma design shows:

![Captura de Pantalla 2022-09-23 a la(s) 8 17 01 a  m](https://user-images.githubusercontent.com/86022690/191968837-d73b0621-a22e-4a45-a357-2430b838a5fd.png)

They should change style when hovered and clicked. 👀